### PR TITLE
First files for Italian Localization

### PR DESCRIPTION
**Pull request by scottaglia**
Some terms are also used commonly in Italian, if translated they would be awkward.

### CODE REVIEW

**Comment by fhemberger** [comment]
@nodejs/iojs-it Please review the changes.

@markainick I just updated the home page with a couple of new labels, added them to the Korean version already: https://github.com/nodejs/new.nodejs.org/commit/4b54ff301e45ac2c1d636e821e8322af25715d95 Please add them to your translation them as well.

File: locale/it/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  current-version: Versione attuale
+  download: Download
+  download-for: Download per
+  other-downloads: Altri Downloads
+  changelog: Changelog
+  api: Documentazione API
+---
+
+Node.js® è un JavaScript runtime costruito sul [motore JavaScript V8 di Chrome](https://developers.google.com/v8/).
+Node.js usa un modello senza blocchi I/O e ad eventi, che lo rendono un framework leggero ed efficiente.
```
Comment (piccoloaiutante) [inline locale/it/index.md]: i would go for "Node.js usa un modello di I/O non bloccante e ad eventi, che lo....."

File: locale/it/site.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            "link": "feed/releases.xml",
+            "text": "Blog Node.js: Releases"
+        },
+        {
+            "link": "feed/vulnerability.xml",
+            "text": "Blog Node.js: Rapporti di Vulnerabilità"
+        },
+        {
+            "link": "feed/tsc-minutes.xml",
+            "text": "Minuti Incontro TSC Node.js"
```
Comment (piccoloaiutante) [inline locale/it/site.json]: i would go for "Minute Incontro Node.js TSC"

File: locale/it/site.json [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        }
+    ],
+    "home": { "text": "Home" },
+    "about": {
+        "link": "about",
+        "text": "About",
+        "governance": {
+            "link": "about/governance",
+            "text": "Governanza"
```
Comment (piccoloaiutante) [inline locale/it/site.json]: i would stay with Governance

File: locale/it/site.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            "link": "docs/faq",
+            "text": "FAQ"
+        },
+        "api": {
+            "link": "/api/",
+            "text": "API"
+        },
+        "knowledge": {
+            "link": "knowledge",
+            "text": "Base di conoscenza"
```
Comment (piccoloaiutante) [inline locale/it/site.json]: i would go for a generic "Informazioni"

File: locale/it/index.md (hunk 1 above)
Comment (lpinca) [inline locale/it/index.md]: I agree with the above but there is a typo, event -> eventi

File: locale/it/site.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    "foundation": {
+        "link": "foundation",
+        "text": "Fondazione",
+        "members": {
+            "link": "foundation/members",
+            "text": "Membri"
+        },
+        "tsc": {
+            "link": "foundation/tsc",
+            "text": "Governanza Tecnica"
```
Comment (piccoloaiutante) [inline locale/it/site.json]: tac literally stay for "Comitato di indirizzo tecnico" we could go for the short form of "Comitato tecnico"

File: locale/it/site.json [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+             "text": "Nelle Notizie"
+        },
+        "announce": {
+            "link": "foundation/announcements",
+            "text": "Annunci"
+        }
+    },
+    "getinvolved": {
+        "link": "get-involved",
+        "text": "Coinvolgiti",
```
Comment (piccoloaiutante) [inline locale/it/site.json]: what about "Come partecipare"

File: locale/it/site.json [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+            "link": "get-involved/contribute",
+            "text": "Contribuire"
+        },
+        "development": {
+            "link": "get-involved/development",
+            "text": "Sviluppo"
+        },
+        "code-and-learn": {
+            "link": "get-involved/code-and-learn",
+            "text": "Codifica + Impara"
```
Comment (piccoloaiutante) [inline locale/it/site.json]: what about "Programma + impara"

**Comment by piccoloaiutante** [comment]
I left few comments for better translation :-)

File: locale/it/site.json (hunk 2 above)
Comment (parro-it) [inline locale/it/site.json]: It'll be better "Minute Incontri Node.js TSC"?

Comment (piccoloaiutante) [inline locale/it/site.json]: yep

File: locale/it/site.json [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+    "download": {
+        "link": "download",
+        "text": "Scarica",
+        "releases": {
+            "link": "download/releases",
+            "text": "Release Precedenti"
+        },
+        "package-manager": {
+            "link": "download/package-manager",
+            "text": "Installa Node.js via package-manager"
```
Comment (lpinca) [inline locale/it/site.json]: I would use "Installa Node.js con un package-manager"

File: locale/it/site.json (hunk 5 above)
Comment (lpinca) [inline locale/it/site.json]: :+1: for "Comitato tecnico"

Comment (parro-it) [inline locale/it/site.json]: Yes, better

File: locale/it/site.json [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+            "link": "foundation/members",
+            "text": "Membri"
+        },
+        "tsc": {
+            "link": "foundation/tsc",
+            "text": "Governanza Tecnica"
+        },
+        "itn": {
+             "link": "foundation/in-the-news",
+             "text": "Nelle Notizie"
```
Comment (lpinca) [inline locale/it/site.json]: "Nelle Notizie" sounds weird I would use only "Notizie"

File: locale/it/site.json (hunk 6 above)
Comment (lpinca) [inline locale/it/site.json]: The above comment is fine, or we could use "Come contribuire"

Comment (piccoloaiutante) [inline locale/it/site.json]: that is the one under

File: locale/it/site.json [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+            "text": "Codifica + Impara"
+        },
+        "events": {
+          "link": "get-involved/events",
+          "text": "Eventi"
+        }
+    },
+    "trademark"  : { "link": "about/trademark", "text": "Marchio" },
+    "security"   : { "link": "security",        "text": "Sicurezza" },
+    "blog"       : { "link": "blog",            "text": "Notizie" },
```
Comment (lpinca) [inline locale/it/site.json]: "Notizie" -> "Blog" ?

Comment (piccoloaiutante) [inline locale/it/site.json]: yes we could use blog

**Comment by scottaglia** [comment]
index.md and site.json updated with your directions

**Comment by fhemberger** [comment]
@nodejs/iojs-it Is this good to go?

File: locale/it/site.json [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+            "text": "Releases"
+        },
+        "resources": {
+            "link": "about/resources",
+            "text": "Risorse"
+        }
+    },
+    "download": {
+        "link": "download",
+        "text": "Scarica",
```
Comment (lpinca) [inline locale/it/site.json]: For consistency I would use "Download" here. This is the only place where "Scarica" is used.

**Comment by lpinca** [comment]
Just a little nit, the rest looks good.

File: locale/it/site.json (hunk 11 above)
Comment (piccoloaiutante) [inline locale/it/site.json]: Yes I would use download as well

**Comment by piccoloaiutante** [comment]
Other than 'download' looks good to me :+1:

File: locale/it/site.json [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+        "releases": {
+            "link": "download/releases",
+            "text": "Release Precedenti"
+        },
+        "package-manager": {
+            "link": "download/package-manager",
+            "text": "Installa Node.js con un package-manager"
```
Comment (lpinca) [inline locale/it/site.json]: @piccoloaiutante do you think that "Installa Node.js con un gestore di pacchetti" would be a better fit here?

Comment (piccoloaiutante) [inline locale/it/site.json]: Yes I think we could translate the entire sentence. Good call ;-)

**Comment by scottaglia** [comment]
site.json updated

**Comment by piccoloaiutante** [comment]
Awesome. Thanks ;-)

**Comment by lpinca** [comment]
@nodejs/website translation is good here. Is this good to merge?

**Comment by ghost** [comment]
LGTM

**Comment by piccoloaiutante** [comment]
:+1:

**Comment by phillipj** [comment]
FYI this broke the build task as `locale/it/site.json` had invalid JSON in it. Just pushed a fix in cd53b08.

**Comment by piccoloaiutante** [comment]
thanks @phillipj

**Comment by lpinca** [comment]
My bad, sorry about that @phillipj.

**Pull request merged**